OTAcceleratorCore/OTScreenCapture.m

```diff
@@ -140,8 +141,8 @@ - (int32_t)stopCapture
     _capturing = NO;
     
     dispatch_sync(_queue, ^{
-        if (_timer) {
-            dispatch_source_cancel(_timer);
+        if (self->_timer) {
```

Comment:
when you use self in a block it is good practice (to avoid reference loops) to capture a weak reference to it.

See: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithBlocks/WorkingwithBlocks.html#//apple_ref/doc/uid/TP40011210-CH8-SW16

Reply:
thanks @robjperez, fixed